core/modules/system/system.install

```diff
@@ -3174,6 +3174,24 @@ function system_update_1074() {
   }
 }
 
+/**
+ * Set a default value for the "Permissions-Policy: interest-cohort" header.
+ *
+ * This will make it so that the Google Federated Learning of Cohorts (FLoC)
+ * feature that gathers user data without cookies is blocked by default, unless
+ * overridden in settings.php.
+ * 
+ * For more information, see the change record at:
+ * https://docs.backdropcms.org/change-records/header-added-by-default-to-disable-floc
+ */
+function system_update_1075() {
+  $config = config('system.core');
+  $config->set('block_interest_cohort', update_variable_get('block_interest_cohort', 1));
```

Comment:
This matches the variable name used in Drupal 7 :+1:

https://git.drupalcode.org/project/drupal/-/commit/eb66609d55ae8e0e70186550a70bd5d73982e783